### PR TITLE
TFA-issue fix - Preempt scrub fix

### PR DESCRIPTION
**Pull request by SrinivasaBharath**
# Description

Jira Task - https://issues.redhat.com/browse/RHCEPHQE-15940

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Comment by SrinivasaBharath** [comment]
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-9UP9M5 - Squid - 60 sec time took  to start  scrub
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-YJ3ATP - Squid - 60 sec time took  to start  scrub
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-YJ3ATP - Squid - 60 sec time took  to start  scrub

**Comment by SrinivasaBharath** [comment]
Reef full suite log- http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-5TJAVH

**Comment by SrinivasaBharath** [comment]
Full suite logs-
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-5TJAVH 
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-JY73X9
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-5R86Q4

File: tests/rados/test_rados_preempt_scrub.py [hunk 1]
```diff
@@ -96,12 +104,18 @@ def run(ceph_cluster, **kw):
...
+            total_time = count * 30
+            log.info(
+                f"The total time to start user initiated scrub is  - {total_time} seconds"
+            )
             # check for the scrub is initiated or it is timeout
             if scrub_status_flag is False:
                 log.error("Scrub is not initiated")
                 return 1
-            time.sleep(10)
+            log.info("Scrub initiated")
```
Comment (pdhiran) [inline tests/rados/test_rados_preempt_scrub.py:117]: add a line to indicate deep-scrub started. minor change.

File: tests/rados/test_rados_preempt_scrub.py [hunk 2]
```diff
@@ -121,10 +135,27 @@ def run(ceph_cluster, **kw):
...
+            # checking the ceph logs
+            mon_log = rados_object.get_journalctl_log(
+                start_time=init_time,
+                end_time=end_time,
+                daemon_type="mon",
+                daemon_id=mon_daemon,
+            )
+            log_mon_count = 0
+
+            log.info(f"The mgr log lines are - {mon_log}")
```
Comment (pdhiran) [inline tests/rados/test_rados_preempt_scrub.py:152]: this should be mon. please update.

**Review by pdhiran — APPROVED** [review]
Looks good. We can make more changes if needed.

**Pull request merged**